helm/prometheus-rules/templates/alerting-rules/capa.management-cluster.rules.yml

```diff
@@ -32,6 +32,7 @@ spec:
       annotations:
         description: '{{`Container {{ $labels.container }} in pod {{ $labels.namespace }}/{{ $labels.pod }} is restarting too often.`}}'
         opsrecipe: container-is-restarting-too-often/
+        dashboard: bdiako8tt1b7kc/capa-agregated-error-logs-for-capa-controllers
```

Comment:
what is the horrible random string?

Reply:
randomly generated dashboard ID

Reply:
You can configure a proper uid instead :)